src/Dependencies/CodeAnalysis.Metadata/CustomDebugInfoEncoder.cs

```diff
@@ -194,7 +194,10 @@ public void AddTupleElementNames(IReadOnlyCollection<(string LocalName, int Slot
                         builder.WriteInt32(info.SlotIndex);
                         builder.WriteInt32(info.ScopeStart);
                         builder.WriteInt32(info.ScopeEnd);
-                        builder.WriteUTF8(info.LocalName);
+                        if (info.LocalName != null)
```

Comment:
@sharwell the two should be equivalent: `WriteUTF8(string.Empty)` should not write any bytes.